src/App/ComplexGeoData.h

```diff
@@ -270,12 +270,12 @@ class AppExport ComplexGeoData: public Base::Persistence, public Base::Handled
      *
      * @return Returns the existing element map.
      */
-    virtual ElementMapPtr resetElementMap(ElementMapPtr elementMap = ElementMapPtr(),
-                                          ElementMapResetPolicy forceEmpty = ForceEmptyMap)
+    virtual ElementMapPtr resetElementMap(ElementMapPtr elementMap = ElementMapPtr()/* ,
+                                          ElementMapResetPolicy forceEmpty = ForceEmptyMap */)
     {
-        if (!elementMap && forceEmpty == ForceEmptyMap) {
-            elementMap = std::make_shared<Data::ElementMap>();
-        }
+        // if (!elementMap && forceEmpty == ForceEmptyMap) {
+        //     elementMap = std::make_shared<Data::ElementMap>();
+        // }

```

Comment:
Agree.  I think you can delete these, not just comment out.  Here, above and below.